src/lasdi/latent_space.py

```diff
@@ -1,175 +1,482 @@
 
-        for i in range(self.n_layers-2):
-            x = self.fcs[i](x) # apply linear layer
-            if (self.use_multihead):
-                x = self.apply_attention(self, x, i)
-            else:
-                x = self.act(x)
+        # Pass x through the network layers (except for the final one, which has no activation 
+        # function).
+        for i in range(self.n_layers - 1):
+            x : torch.Tensor = self.layers[i](x)   # apply linear layer
+            x : torch.Tensor = self.act(x)      # apply activation
 
-        x = self.fcs[-1](x)
+        # Apply the final (output) layer.
+        x = self.layers[-1](x)
 
+        # If the reshape_index is -1, then we need to reshape the output before returning. 
         if (self.reshape_index == -1):
-            # we use torch.Tensor.view instead of torch.Tensor.reshape,
-            # in order to avoid data copying.
+            # In this case, we need to split the last dimension of x, the output of the final
+            # layer, to match the reshape_shape. This is precisely what the line below does. Note
+            # that we use torch.Tensor.view instead of torch.Tensor.reshape in order to avoid data 
+            # copying. 
             x = x.view(list(x.shape[:-1]) + self.reshape_shape)
 
+        # All done!
         return x
     
-    def apply_attention(self, x, act_idx):
-        x = x.unsqueeze(1)  # Add sequence dimension for attention
-        x, _ = self.act[act_idx](x, x, x) # apply attention
-        x = x.squeeze(1)  # Remove sequence dimension
-        return x
-    
-    def init_weight(self):
+
+
+    def init_weight(self) -> None:
+        """
+        This function initializes the weight matrices and bias vectors in self's layers.
+
+
+        -------------------------------------------------------------------------------------------
+        Arguments
+        -------------------------------------------------------------------------------------------
+
+        None!
+
+        
+        -------------------------------------------------------------------------------------------
+        Returns
+        -------------------------------------------------------------------------------------------
+
+        Nothing!
+        """
+
         # TODO(kevin): support other initializations?
-        for fc in self.fcs:
-            torch.nn.init.xavier_uniform_(fc.weight)
+        for layer in self.layers:
+            torch.nn.init.xavier_uniform_(layer.weight)
+            torch.nn.init.zeros_(layer.bias)
+        
+        # All done!
         return
 
+
+
+# -------------------------------------------------------------------------------------------------
+# Autoencoder class
+# -------------------------------------------------------------------------------------------------
+
 class Autoencoder(torch.nn.Module):
+    def __init__(self, physics : Physics, config : dict) -> None:
+        """
```

Comment:
Currently this is somehow not parsed by API. I found that we need to turn on the option. You can wait until I push a new PR for it, or it'd be great if you can add one liner in line 28 of `docs/source/conf.py`:
```
autoapi_python_class_content = 'both'
```